src/app/modules/group/pages/group-edit/group-edit.component.ts

```diff
@@ -90,13 +90,14 @@ export class GroupEditComponent implements OnDestroy {
       this.groupId,
       this.getGroupChanges(),
     ).subscribe(
-      _status => {
+      _result => {},
```

Comment:
but what result? it is observable of void. isn't it?

Reply:
fixed